docs/gettingstarted/quickstart.rst

```diff
@@ -0,0 +1,293 @@
+         Install the flocker-cli client on your Linux machine:
+         
+         .. code-block:: console
+         
+            you@laptop:~$ sudo yum install -y @buildsys-build python python-devel python-virtualenv && \
+              virtualenv flocker-tutorial && \
+              flocker-tutorial/bin/pip install --upgrade pip && \
+              flocker-tutorial/bin/pip install --quiet https://storage.googleapis.com/archive.clusterhq.com/downloads/flocker/Flocker-0.3.2-py2-none-any.whl && source flocker-tutorial/bin/activate
+
+      .. empty-div:: arrow-down center-block invisible
+
+   .. parallel::
+
+      Installation
+      ------------
+
+      .. mobile-label::
+
+         Live
+
+      .. noscript-content::
+
+         Vagrant
+         ^^^^^^^
+
+         Simulate a Flocker cluster with virtual machines on your laptop (requires `Vagrant <http://www.vagrantup.com/downloads>`_, `VirtualBox <https://www.virtualbox.org/wiki/Downloads>`_):
+
+         .. code-block:: console
+
+            you@laptop:~$ git clone \
+              https://github.com/clusterhq/vagrant-flocker && \
+              cd vagrant-flocker && \
+              vagrant up
+
+         AWS
+         ^^^
+
+         Please see our separate `AWS install instructions <http://docs.clusterhq.com/en/latest/gettingstarted/installation.html#using-amazon-web-services>`_ to get started.
+
+      .. tabs::
+
+         Vagrant
+         ^^^^^^^
+
+         Simulate a Flocker cluster with virtual machines on your laptop (requires `Vagrant <http://www.vagrantup.com/downloads>`_, `VirtualBox <https://www.virtualbox.org/wiki/Downloads>`_):
+
+         .. code-block:: console
+
+            you@laptop:~$ git clone \
+              https://github.com/clusterhq/vagrant-flocker && \
+              cd vagrant-flocker && \
+              vagrant up
+
+         AWS
+         ^^^
+
+         Please see our separate `AWS install instructions <http://docs.clusterhq.com/en/latest/gettingstarted/installation.html#using-amazon-web-services>`_ to get started.
+
+
+      .. empty-div:: arrow-down arrow-offset center-block
+
+.. tutorial-step::
+
+   Step 2: Deploying a demo app
+   ============================
+   
+   .. tutorial-step-condensed::
+
+      You should have flocker-cli installed on your laptop and flocker-node installed on some servers: either VMs on your laptop, or real instances on cloud infrastructure.
+      Now you can try our simple tutorial: a Python web application and a Redis server.
+
+      .. code-block:: console
+
+         you@laptop:~$ git clone https://github.com/clusterhq/flocker-quickstart
+         you@laptop:~$ cd flocker-quickstart
+
+      fig.yml
+      -------
+
+      .. code-block:: yaml
+
+         web:
+           image: clusterhq/flask
+           links:
+            - "redis:redis"
+           ports:
+            - "80:80"
+         redis:
+           image: dockerfile/redis
+           ports:
+            - "6379:6379"
+           volumes: ["/data"]
```

Comment:
Please file a ticket for keeping this code block up-to-date with the content in `flocker-quickstart`.